### ISSUE TITLE
Themes Stop Working After an Hour or So

### ISSUE THREAD

**Issue by isaacdew** [mhoofman/wordpress-heroku#118]
I've tried using multiple themes and each time the theme goes offline and the WordPress dashboard warns that an error occured with the theme. Since the WordPress dashboard still works I assume that the issue is incompatibility between the theme and this modified version of WordPress.

**Comment by lifenautjoe** [comment]
No, this is because Heroku machines are ephemeral, meaning any plugin or theme you install in the machine will dissapear after a while. If you want to persist your themes or plugins do it on the repository and then deploy.
